### DISCUSSION TITLE
How to make upper/ lower case not matter?

### DISCUSSION THREAD

**Discussion by jdaviescoates** [YOURLS/YOURLS#3551] [Q&A] [answered]
I've got the following shortlink set-up:

https://ud.coop/bestofbridport

And I need https://ud.coop/BESTOFBRIDPORT to work as well.

I can't add BESTOFBRIDPORT as another shortlink as it's already in use.

Is there a way I can get YOURLS to ignore upper/ lower case so that https://ud.coop/BESTOFBRIDPORT works exactly the same as https://ud.coop/bestofbridport ?

Thanks in advance for your help.

**Comment by ozh** [comment]
https://github.com/YOURLS/YOURLS/issues/2333 for example

**Comment by jdaviescoates** [comment]
Thanks @ozh

I *think* what you're saying is that is I update from base 36 to base 62 then both https://ud.coop/bestofbridport and https://ud.coop/BESTOFBRIDPORT would both work and redirect to the same link, is that right?

If so, how do I do that as I have no idea?! 

Thanks again,

Josef.

**Comment by dgw** [comment, marked as answer]
Explore the three "case insensitive" plugin options listed here: https://github.com/YOURLS/awesome#c

The most recent of those (https://github.com/IMExperts/yourls-case-insensitive) assumes base 62 charset.

The oldest one under the "case insensitive" moniker (https://github.com/seandrickson/YOURLS-Case-Insensitive/) assumes base 36.

There is also @ozh's own "Force Lowercase" plugin (https://github.com/YOURLS/force-lowercase).

_**Note:** I haven't personally tested any of these. Try them out on a test instance before adding them to your main one, or at least take a backup of your existing YOURLS install prior to messing with these plugins so you can roll back if something breaks._

**Comment by jdaviescoates** [comment]
Thanks @dgw this one worked a treat https://github.com/IMExperts/yourls-case-insensitive (although I tried that before spotting your edits that added the "Force Lowercase" plugin so I may try that one too/ instead given it's more recently updated and by @ozh
